gsitemap.php

```diff
@@ -629,6 +631,21 @@ public function createSitemap($id_shop = 0)
         die();
     }
 
+    /**
+     * Loops through created XML files and fix the indentation to comply with Google requirements
+     */
+    public function indentAllXMLFiles()
+    {
+        $files = Db::getInstance()->ExecuteS('SELECT * FROM `' . _DB_PREFIX_ . 'gsitemap_sitemap` WHERE id_shop = ' . (int) $this->context->shop->id);
```

Comment:
I'm not sure this works with multistore 🤔 because of `$this->context->shop->id`

Reply:
it will not works on multistore :thinking: